src/windows.rs

```diff
@@ -6,23 +6,29 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use core::ffi::c_void;
+use core::mem::MaybeUninit;
+use core::num::NonZeroU32;
+use core::ptr;
+
 use crate::Error;
-use core::{ffi::c_void, num::NonZeroU32, ptr};
 
 const BCRYPT_USE_SYSTEM_PREFERRED_RNG: u32 = 0x00000002;
 
 #[link(name = "bcrypt")]
 extern "system" {
     fn BCryptGenRandom(
         hAlgorithm: *mut c_void,
-        pBuffer: *mut u8,
+        pBuffer: *mut MaybeUninit<u8>,
```

Comment:
T and MaybeUninit<T> is equal for FFI purposes: https://doc.rust-lang.org/stable/std/mem/union.MaybeUninit.html#layout